repository-data/application/src/main/resources/hcm-content/content/urlrewriter/medical/gpnrotomedicalgpredirects.yaml

```diff
@@ -68,6 +68,8 @@
       permanent-redirect\" last=\"true\">https://medical.hee.nhs.uk/medical-training-recruitment/medical-specialty-training/general-practice-gp/choose-gp/choose-gp-career-stories/dr-haider-ali</to>\r\
       \n</rule>\r\n\r\n<rule>\r\n  <condition name=\"host\" operator=\"equal\">gprecruitment.hee.nhs.uk</condition>\r\
       \n  <from>^(.*)$</from>\r\n  <to type=\"permanent-redirect\" last=\"true\">https://medical.hee.nhs.uk/medical-training-recruitment/medical-specialty-training/general-practice-gp</to>\r\
+      \n</rule>\r\n\r\n<rule>\r\n  <condition name=\"host\" operator=\"equal\">gprecruitment-prod.hee.nhs.uk</condition>\r\
```

Comment:
@sankar-shunmuga - do we need to add other `gprecruitment.hee.nhs.uk` redirects as well for `gprecruitment-prod.hee.nhs.uk` ? thanks

Reply:
Email sent to Mark Baldwin. If he comes back, we can create a separate ticket